docs/samples/client/kfserving_sdk_sample.ipynb

```diff
@@ -77,9 +78,34 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 19,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
Not sure if we should clean up the cell outputs for easy reviewing, this is optional :-)

Reply:
cleared the output